addons/web/static/src/xml/file_upload_progress_bar.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+<t t-name="web.ProgressCard">
```

Comment:
empty line before

Reply:
isn't it better to split card and bar in 2 different XML files? Or rename file so that we expect seeing both of them here